src/components/layouts/layout.css

```diff
@@ -3,6 +3,18 @@ body {
   padding: 0px;
 }
 
+/* Class definition */
+
+.onHoverDisplay {
+  height: 100%;
+  width: 100%;
+  display: none;
+}
+.onHoverHide {
```

Comment:
Generally speaking, I'm not sure I love having `width` and `height` on these `hover` classes - if I was using a `hover` class, I'm mostly expecting it to have to deal with making something visible/invisible based on hover state, not have `width`/`height` side effects...

Maybe define a different class for setting the `height` & `width`, and just use the hover classes to control the display state?